test/browser_tests/spec_suites/organisms/header.js

```diff
@@ -135,11 +134,31 @@ describe( 'Header', function() {
               expect( _dom.overlay.isDisplayed() ).toBe( true );
             } );
         } );
+
+        describe( 'then click search', function() {
+          it( 'should show the search and hide menu', function() {
+            browser.driver.actions().click( _dom.megaMenuTrigger ).perform();
+            browser.driver.wait( _dom.megaMenuContent.isDisplayed() );
+            browser.driver.actions().click( _dom.globalSearchTrigger ).perform()
+              .then( function() {
+                expect( _dom.globalSearch.isDisplayed() ).toBe( true );
+
+              } );
+          } );
+        } );
+      } );
+
+      describe( 'click search, then click mega menu', function() {
+        it( 'should show the megamenu and hide search', function() {
+          browser.driver.actions().click( _dom.globalSearchTrigger ).perform();
+          browser.driver.wait( _dom.megaMenuContent.isDisplayed() );
```

Comment:
I was looking at the wrong lines, now I see it. So I tried testing something, it looks like 

```expect( _dom.globalSearch.isDisplayed() ).toBe( true );
 expect( _dom.megaMenuContent.isDisplayed() ).toBe( true );
```

always resolve as true when I test on smaller screens. I haven't been able to write a test where that's false. I'm going to read documentation to see if there's another test I can use, but does anyone have ideas?